From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Mon, 1 May 2017 03:37:38 +0000
Subject: [PATCH] config: no Atom

---
 drivers/platform/x86/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/platform/x86/Kconfig b/drivers/platform/x86/Kconfig
index 0581a54..fca3775 100644
--- a/drivers/platform/x86/Kconfig
+++ b/drivers/platform/x86/Kconfig
@@ -1399,6 +1399,6 @@ config INTEL_TELEMETRY
 endif # X86_PLATFORM_DEVICES
 
 config PMC_ATOM
-       def_bool y
+       def_bool n
        depends on PCI
        select COMMON_CLK
-- 
https://clearlinux.org

