From 14739f976efeabc3158e4f2b6c1a3cb2fb83ff14 Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Mon, 1 May 2017 03:37:38 +0000
Subject: [PATCH 119/125] config: no Atom

---
 drivers/platform/x86/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/platform/x86/Kconfig b/drivers/platform/x86/Kconfig
index ac4d48830415..0a51e7b3d281 100644
--- a/drivers/platform/x86/Kconfig
+++ b/drivers/platform/x86/Kconfig
@@ -1221,6 +1221,6 @@ config INTEL_CHTDC_TI_PWRBTN
 endif # X86_PLATFORM_DEVICES
 
 config PMC_ATOM
-       def_bool y
+       def_bool n
        depends on PCI
        select COMMON_CLK
-- 
2.18.0

