From f38b9287af7c5905c481d71d8184c49fdf81951c Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Sun, 14 May 2017 23:53:08 +0000
Subject: [PATCH 1001/1003] acpi: verbose

---
 drivers/acpi/acpica/nsxfeval.c | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/drivers/acpi/acpica/nsxfeval.c b/drivers/acpi/acpica/nsxfeval.c
index c944ff5c9c3d..20e4c2aa9f5b 100644
--- a/drivers/acpi/acpica/nsxfeval.c
+++ b/drivers/acpi/acpica/nsxfeval.c
@@ -43,7 +43,7 @@
  */
 
 #define EXPORT_ACPI_INTERFACES
-
+#include <linux/acpi.h>
 #include <acpi/acpi.h>
 #include "accommon.h"
 #include "acnamesp.h"
@@ -191,6 +191,8 @@ acpi_evaluate_object(acpi_handle handle,
 		return_ACPI_STATUS(AE_NO_MEMORY);
 	}
 
+	acpi_handle_err(handle, "acpi_evaluate_object: start %s\n", pathname);
+
 	/* Convert and validate the device handle */
 
 	info->prefix_node = acpi_ns_validate_handle(handle);
@@ -456,6 +458,8 @@ acpi_evaluate_object(acpi_handle handle,
 	}
 
 	ACPI_FREE(info);
+	//printk("acpi_evaluate_object: end %s\n", pathname);
+
 	return_ACPI_STATUS(status);
 }
 
-- 
2.13.2

