From feb4cce0845b543997392346e592d4da5a0065d5 Mon Sep 17 00:00:00 2001
From: Arjan van de Ven <arjan@linux.intel.com>
Date: Sun, 14 May 2017 23:53:08 +0000
Subject: [PATCH 1003/1003] acpi: print ADR

---
 drivers/acpi/bus.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/acpi/bus.c b/drivers/acpi/bus.c
index 784bda663d16..d2cd389beadb 100644
--- a/drivers/acpi/bus.c
+++ b/drivers/acpi/bus.c
@@ -98,6 +98,7 @@ acpi_status acpi_bus_get_status_handle(acpi_handle handle,
 	acpi_status status;
 
 	status = acpi_evaluate_integer(handle, "_STA", NULL, sta);
+	printk("_STA called, ADR is %p\n", device->address);
 	if (ACPI_SUCCESS(status))
 		return AE_OK;
 
@@ -124,6 +125,7 @@ int acpi_bus_get_status(struct acpi_device *device)
 		return -ENODEV;
 
 	acpi_set_device_status(device, sta);
+	printk("_STA called, ADR is %p\n", device->pnp.bus_address);
 
 	if (device->status.functional && !device->status.present) {
 		ACPI_DEBUG_PRINT((ACPI_DB_INFO, "Device [%s] status [%08x]: "
-- 
2.13.2

