--- linux-4.11.1/include/acpi/acpi_bus.h~	2017-05-14 12:06:16.000000000 +0000
+++ linux-4.11.1/include/acpi/acpi_bus.h	2017-05-14 23:14:52.383427356 +0000
@@ -361,6 +361,7 @@
 	struct list_head wakeup_list;
 	struct list_head del_list;
 	struct acpi_device_status status;
+	int status_valid;
 	struct acpi_device_flags flags;
 	struct acpi_device_pnp pnp;
 	struct acpi_device_power power;
--- linux-4.11.1/include/acpi/acpi_bus.h~	2017-05-14 23:14:52.000000000 +0000
+++ linux-4.11.1/include/acpi/acpi_bus.h	2017-05-14 23:16:03.039420426 +0000
@@ -444,6 +444,7 @@
 static inline void acpi_set_device_status(struct acpi_device *adev, u32 sta)
 {
 	*((u32 *)&adev->status) = sta;
+	adev->status_valid = 1;
 }
 
 static inline void acpi_set_hp_context(struct acpi_device *adev,
--- linux-4.11.1/drivers/acpi/scan.c~	2017-05-14 12:06:16.000000000 +0000
+++ linux-4.11.1/drivers/acpi/scan.c	2017-05-14 23:17:42.214033082 +0000
@@ -1833,7 +1833,8 @@
 	if (ACPI_SUCCESS(acpi_bus_get_ejd(device->handle, &ejd)))
 		register_dock_dependent_device(device, ejd);
 
-	acpi_bus_get_status(device);
+	if (!device->status_valid)
+		acpi_bus_get_status(device);
 	/* Skip devices that are not present. */
 	if (!acpi_device_is_present(device)) {
 		acpi_device_clear_enumerated(device);
