--- linux-4.11.1/drivers/acpi/acpica/nsxfeval.c~	2017-05-14 12:06:16.000000000 +0000
+++ linux-4.11.1/drivers/acpi/acpica/nsxfeval.c	2017-05-14 14:57:17.643061945 +0000
@@ -191,6 +191,8 @@
 		return_ACPI_STATUS(AE_NO_MEMORY);
 	}
 
+	acpi_handle_err(handle, "acpi_evaluate_object: start %s\n", pathname);
+
 	/* Convert and validate the device handle */
 
 	info->prefix_node = acpi_ns_validate_handle(handle);
@@ -456,6 +458,8 @@
 	}
 
 	ACPI_FREE(info);
+	//printk("acpi_evaluate_object: end %s\n", pathname);
+
 	return_ACPI_STATUS(status);
 }
 
--- linux-4.11.1/drivers/acpi/acpica/nsxfeval.c~	2017-05-14 15:15:37.000000000 +0000
+++ linux-4.11.1/drivers/acpi/acpica/nsxfeval.c	2017-05-14 15:32:38.773424125 +0000
@@ -43,7 +43,7 @@
  */
 
 #define EXPORT_ACPI_INTERFACES
-
+#include <linux/acpi.h>
 #include <acpi/acpi.h>
 #include "accommon.h"
 #include "acnamesp.h"
