--- linux-4.10.10/mm/vmstat.c~	2017-04-12 10:43:04.000000000 +0000
+++ linux-4.10.10/mm/vmstat.c	2017-05-01 01:00:13.299992480 +0000
@@ -1549,7 +1549,7 @@
 #ifdef CONFIG_SMP
 static struct workqueue_struct *vmstat_wq;
 static DEFINE_PER_CPU(struct delayed_work, vmstat_work);
-int sysctl_stat_interval __read_mostly = HZ;
+int sysctl_stat_interval __read_mostly = 8 * HZ;
 
 #ifdef CONFIG_PROC_FS
 static void refresh_vm_stats(struct work_struct *work)
